deploy-dotorg.sh

```diff
@@ -11,96 +11,198 @@ if [[ -z "$SVN_PASSWORD" ]] && [[ $1 != "preview" ]]; then
 	exit 1
 fi
 
+# These themes either:
+# 	are not (and not expected to be) deployed to wporg
+# 	have been deployed to wporg, however they are not expected to be updated
+# Note that all classic themes (those without a theme.json file) are ALSO ignored
+declare -a THEMES_TO_IGNORE=(
+        'aldente'
+        'allez'
+        'alter'
+        'ames'
+        'antonia'
+        'appleton'
+        'arbutus'
+        'artly'
+        'assembler'
+        'attar'
+        'awburn'
+        'azur'
+        'barnett'
+        'barnsbury23'
+        'beep'
+        'bennett'
+        'blank-canvas-3'
+        'block-canvas'
+        'blogorama'
+        'boxedbio'
+        'calvin'
+        'calyx'
+        'chanson'
+        'cortado'
+        'covr'
+        'craftfully'
+        'creatio-2'
+        'creatio'
+        'curriculum'
+        'dorna'
+        'dos'
+        'entry'
+        'epi'
+        'erma'
+        'eventual'
+        'exmoor'
+        'farrow'
+        'foam'
+        'grammerone'
+        'hall'
+        'hari'
+        'heiwa'
+        'ici'
+        'indice'
+        'infield'
+        'iotix'
+        'issue'
+        'jackson'
+        'kansei'
+        'kaze'
+        'kingsley'
+        'kiosko'
+        'lativ'
+        'loic'
+        'lois'
+        'luce'
+        'lynx'
+        'marl'
+        'masu'
+        'meraki'
+        'messagerie'
+        'montagna'
+        'mpho'
+        'muscat'
+        'mysa'
+        'nested'
+        'nook'
+        'organizer'
+        'otis'
+        'overlaid'
+        'perenne'
+        'pieria'
+        'poesis'
+        'pomme'
+        'programme'
+        'raw'
+        'reverie'
+        'ritratto'
+        'russell'
+        'screenplay'
+        'shhh'
+        'snd'
+        'spearhead-blocks'
+        'spiel'
+        'stage'
+        'startfit'
+        'sten'
+        'strand'
+        'sunderland'
+        'tenaz'
+        'texty'
+        'the-jazzers'
+        'the-menu'
+        'tu'
+        'twentytwentytwo-blue'
+        'twentytwentytwo-mint'
+        'twentytwentytwo-pink'
+        'twentytwentytwo-red'
+        'twentytwentytwo-swiss'
+        'verso'
+        'vetro'
+        'winkel'
+        'xanadu'
+)
+
 rm -rf ./deploy
 
-# Look into all the themes we expect to deploy
-
-declare -a THEMES_TO_DEPLOY=(
-	"adventurer"
-	"archeo"
-	"bibimbap"
-	"bitacora"
-	"blockbase"
-	"course"
-	"ctlg"
-	"disco"
-	"geologist"
-	"george-lois"
-	"hey"
-	"lettre"
-	"lineup"
-	"livro"
-	"mayland-blocks"
-	"paimio"
-	"pendant"
-	"pixl"
-	"poema"
-	"quadrat"
-	"rainfall"
-	"remote"
-	"seedlet-blocks"
-	"skatepark"
-	"stewart"
-	"storia"
-	"upsidedown"
-	"vivre"
-	"zoologist"
-)
+# Do things for all of the themes
+for THEME_SLUG in */ ; do
 
-for THEME_SLUG in ${THEMES_TO_DEPLOY[@]} ; do
+	# Skip any classic themes (assuming that none of our classic themes use a theme.json file)
+	if ! test -f "./${THEME_SLUG}/theme.json"; then
```

Comment:
Yeah, I guess that's what CORE is looking for to determine eligibility (though that also takes into account a theme's PARENT too right?  That would be harder to check and I'm not sure that it would matter for our purposes.)